<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="GLPainel" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>561</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="labelX">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>260</y>
      <width>55</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;x&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelY">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>290</y>
      <width>55</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;y&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelZ">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>320</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;z&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxMalha">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>121</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 1px;
border-style: solid;</string>
    </property>
    <property name="title">
     <string>Malhas</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="checkBoxMalhaXZ">
       <property name="styleSheet">
        <string notr="true">border-width: 0px;
font-weight: nomal;</string>
       </property>
       <property name="text">
        <string>xz</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="checkBoxMalhaXY">
       <property name="styleSheet">
        <string notr="true">border-width: 0px;
font-weight: nomal;</string>
       </property>
       <property name="text">
        <string>xy</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="checkBoxMalhaYZ">
       <property name="styleSheet">
        <string notr="true">border-width: 0px;
font-weight: nomal;</string>
       </property>
       <property name="text">
        <string>yz</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBoxEixos">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>130</y>
      <width>121</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 1px;
border-style: solid;</string>
    </property>
    <property name="title">
     <string>Eixos</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="checkBoxEixoY">
       <property name="styleSheet">
        <string notr="true">border-width: 0px;
font-weight: nomal;</string>
       </property>
       <property name="text">
        <string>y</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="checkBoxEixoX">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">border-width: 0px;
font-weight: nomal;</string>
       </property>
       <property name="text">
        <string>x</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="checkBoxEixoZ">
       <property name="styleSheet">
        <string notr="true">border-width: 0px;
font-weight: nomal;</string>
       </property>
       <property name="text">
        <string>z</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>widget</zorder>
   <zorder>groupBoxMalha</zorder>
   <zorder>groupBoxEixos</zorder>
   <zorder>labelX</zorder>
   <zorder>labelY</zorder>
   <zorder>labelZ</zorder>
   <zorder>groupBoxMalha</zorder>
   <zorder>groupBoxEixos</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>721</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionAbrirArquivo"/>
    <addaction name="actionSair"/>
   </widget>
   <widget class="QMenu" name="menuSobre">
    <property name="title">
     <string>Exibir</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSobre_2">
    <property name="title">
     <string>Ajuda</string>
    </property>
    <addaction name="actionManual"/>
    <addaction name="actionSobre"/>
   </widget>
   <addaction name="menuArquivo"/>
   <addaction name="menuSobre"/>
   <addaction name="menuSobre_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbrirArquivo">
   <property name="text">
    <string>Abrir arquivo</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="actionManual">
   <property name="text">
    <string>Manual</string>
   </property>
  </action>
  <action name="actionSobre">
   <property name="text">
    <string>Sobre</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLPainel</class>
   <extends>QWidget</extends>
   <header>glpainel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
